src/components/20-molecules/datepicker/unit.test.js

```diff
@@ -64,20 +64,118 @@ describe('Datepicker unit tests', () => {
     it('should set class variables if output:true', () => {
       // init values
       AXADatepicker.prototype.outputdate = null;
-      AXADatepicker.prototype._userSelectedDate = null;
+      AXADatepicker.prototype._selectedDate = null;
 
       AXADatepicker.prototype.initDate(new Date(), { output: true });
 
       expect(AXADatepicker.prototype.outputdate).not.toBe(null);
-      expect(AXADatepicker.prototype._userSelectedDate).not.toBe(null);
+      expect(AXADatepicker.prototype._selectedDate).not.toBe(null);
     });
     it('should set class variables if tentative:false', () => {
       // init values
-      AXADatepicker.prototype._preselectedDate = null;
+      AXADatepicker.prototype._selectedDate = null;
 
       AXADatepicker.prototype.initDate(new Date(), { tentative: false });
 
-      expect(AXADatepicker.prototype._userSelectedDate).not.toBe(null);
+      expect(AXADatepicker.prototype._selectedDate).not.toBe(null);
+    });
+  });
+
+  describe('showNextMonth()', () => {
```

Comment:
not a big fan of unit tests for UI component implementation details; if the above e2e tests are more comprehensive, you could skip these below (and you would be immune to changes of function names and signature, as suggested above).